modules/admin/src/resources/scss/components/_luyasearch.scss

```diff
@@ -1,78 +1,104 @@
 .luyasearch {
-    position: absolute;
-    display: block;
-    width: 640px;
-    max-width: 100%;
-    top: 0;
-    bottom: 0;
-    background: white;
-    z-index: z-index(luyasearch);
-    padding-left: 62px;
-    transition: .25s left ease-in-out;
-    box-shadow: 0 0 10px -2px map-get($grays, 800);
-    overflow-y: auto;
 
-    @include media-breakpoint-up(lg) {
-        width: 70%;
-    }
+    display: none;
 
-    &-closed {
-        left: -100%;
-    }
+    @media (min-width: $mobile-nav-breakpoint) {
+        position: absolute;
+        display: block;
+        width: 640px;
+        max-width: 100%;
+        top: 0;
+        bottom: 0;
+        z-index: z-index(luyasearch);
+        padding-left: 62px;
+        transition: .25s left ease-in-out;
+        overflow-y: auto;
+
+        @include media-breakpoint-up(lg) {
+            width: 70%;
+        }
 
-    &-open {
-        left: 0;
-    }
+        &-closed {
+            left: -100%;
+        }
 
-    &-inner {
-        padding: $content-padding-x;
-    }
+        &-open {
+            left: 0;
+        }
 
-    &-toggled {
-        padding-left: 200px;
+        &-toggled {
+            padding-left: 200px;
+
+            .luyasearch-form {
+                @include media-breakpoint-up(lg) {
+                    width: calc(70% - 200px);
+                }
+            }
+        }
     }
 }
 
-.luyasearch-form {
+.luyasearch-inner {
+
+    background: $white;
+    min-height: 100%;
+    padding: 72px $content-padding-y $content-padding-x $content-padding-y;
     position: relative;
-    background: map-get($grays, 100);;
-    padding: $content-padding-x;
+
+}
+
+.luyasearch-form {
+
+    top: 0;
+    width: 440px;
+    background-color: color(gray);
+    padding: $content-padding-x $content-padding-y $content-padding-x 0;
+    margin-bottom: 0;
+    border-bottom-right-radius: $border-radius;
+    position: fixed;
+    z-index: z-index(toast);
```

Comment:
z-index issue

Reply:
its changed, search is having own z-index element now